src/storage/src/render/upsert.rs

```diff
@@ -137,7 +138,10 @@ pub(crate) fn upsert<G: Scope, O: timely::ExchangeData + Ord>(
     source_config: crate::source::RawSourceCreationConfig,
     instance_context: &StorageInstanceContext,
     dataflow_paramters: &crate::internal_control::DataflowParameters,
-) -> Collection<G, Result<Row, DataflowError>, Diff>
+) -> (
+    Collection<G, Result<Row, DataflowError>, Diff>,
+    Stream<G, (WorkerId, OutputIndex, HealthStatusUpdate)>,
```

Comment:
doc comment should describe what the return value is

Reply:
Added